code/modules/mob/living/simple_animal/abnormality/aleph/melting_love.dm

```diff
@@ -301,3 +323,134 @@
+		deltimer(timer2)
+		timer2 = null
+	timer1 = addtimer(CALLBACK(src, PROC_REF(Reduce)), 10 SECONDS, TIMER_STOPPABLE)
+	timer2 = addtimer(CALLBACK(src, PROC_REF(Reduce)), 20 SECONDS, TIMER_STOPPABLE)
+
+
+/obj/effect/decal/cleanable/melty_slime/proc/Reduce()
+	state -= 1
+	icon_state = "melty_slime[state]"
+	update_icon()
+
+/obj/effect/decal/cleanable/melty_slime/process(delta_time)
+	if(world.time > duration)
+		Remove()
+
+/obj/effect/decal/cleanable/melty_slime/proc/Remove()
+	STOP_PROCESSING(SSobj, src)
+	animate(src, time = (5 SECONDS), alpha = 0)
+	QDEL_IN(src, 5 SECONDS)
+
+/obj/effect/decal/cleanable/melty_slime/proc/streak(list/directions, mapload=FALSE)
+	set waitfor = FALSE
+	var/direction = pick(directions)
+	for(var/i in 0 to pick(0, 200; 1, 150; 2, 50; 3, 17; 50)) //the 3% chance of 50 steps is intentional and played for laughs.
+		if (!mapload)
+			sleep(2)
+		if(!step_to(src, get_step(src, direction), 0))
+			break
+
+/obj/effect/decal/cleanable/melty_slime/Crossed(atom/movable/AM)
+	. = ..()
+	if(!isliving(AM))
+		return FALSE
+	if(is_type_in_list(AM, slime_types, FALSE))
+		return
+	var/mob/living/L = AM
+	L.apply_slimed()
+
+/obj/effect/gibspawner/generic/silent/melty_slime
+	gibtypes = list(/obj/effect/decal/cleanable/melty_slime)
+	gibamounts = list(3)
+
+/obj/effect/gibspawner/generic/silent/melty_slime/Initialize()
+	if(!gibdirections.len)
+		gibdirections = list(list(WEST, NORTHWEST, SOUTHWEST, NORTH))
+	. = ..()
+	return
+
+/datum/status_effect/melty_slimed
+	id = "melty_slimed"
+	status_type = STATUS_EFFECT_REPLACE
+	alert_type = /atom/movable/screen/alert/status_effect/melty_slimed
+	duration = 10 SECONDS // Hits 5 times
+	tick_interval = 2 SECONDS
+
+/atom/movable/screen/alert/status_effect/melty_slimed
+	name = "Acidic Goo"
+	desc = "Slime is stuck to your skin, slowing you down and dealing BLACK damage!"
+	icon = 'ModularTegustation/Teguicons/status_sprites.dmi'
+	icon_state = "slimed"
+
+/datum/status_effect/melty_slimed/tick()
+	. = ..()
+	if(!isliving(owner))
+		return
+	var/mob/living/L = owner
+	L.apply_damage(10, BLACK_DAMAGE, null, L.run_armor_check(null, BLACK_DAMAGE), spread_damage = TRUE)
+	owner.playsound_local(owner, 'sound/effects/wounds/sizzle2.ogg', 25, TRUE)
+	if(!ishuman(L))
+		return
+	if((L.sanityhealth <= 0) || (L.health <= 0))
+		var/turf/T = get_turf(L)
+		new /mob/living/simple_animal/hostile/slime(T)
+		L.gib(TRUE, TRUE, TRUE)
+
+/datum/status_effect/melty_slimed/on_apply()
+	owner.add_movespeed_modifier(/datum/movespeed_modifier/slimed)
+	owner.playsound_local(owner, 'sound/abnormalities/meltinglove/ranged_hit.ogg', 50, TRUE)
+	return ..()
+
+/datum/status_effect/melty_slimed/on_remove()
+	owner.remove_movespeed_modifier(/datum/movespeed_modifier/slimed)
+	return ..()
+
+//Mob Proc
+/mob/living/proc/apply_slimed()
+	var/datum/status_effect/melty_slimed/S = src.has_status_effect(/datum/status_effect/melty_slimed)
+	if(!S)
+		src.apply_status_effect(STATUS_EFFECT_SLIMED)
+	else
+		S.refresh()
```

Comment:
No. If you want it to REFRESH then make status_type be STATUS_EFFECT_REFRESH and just _apply the status effect normally_. This proc is redundant.